src/lib.rs

```diff
@@ -80,6 +82,8 @@ mod codegen {
     include!(concat!(env!("OUT_DIR"), "/codegen.rs"));
 }
 
+pub use features::{RustTarget, LATEST_STABLE_RUST, RUST_TARGET_STRINGS};
```

Comment:
Huh, well there are no warnings in the travis ci compilation (and I guess we deny(warnings) anyhow).